public/docs/_examples/attribute-directives/ts/src/app/app.component.html

```diff
@@ -1,5 +1,4 @@
-<!-- #docregion -->
-<!-- #docregion v2 -->
+<!-- #docregion v2, -->
```

Comment:
Ok, I'll refrain from combining them in the future (I just find excessive docregion lines distracting when trying to read the source).